File: Day2/Solution.py

class Solution:
    def transpose(self, matrix: List[List[int]]) -> List[List[int]]:
        transpose = []
        for j in range(len(matrix[0])):
            row = []
            for i in range(len(matrix)):
                row.append(matrix[i][j])
            transpose.append(row)
        return transpose

File: Day2/Solution.java
package Day2;

class Solution {
    public int[][] transpose(int[][] matrix) {
        int[][] transpose = new int[(matrix[0].length)][(matrix.length)];
        for(int j = 0;j<(matrix[0].length);j++){
            for(int i = 0;i<(matrix.length);i++){
                transpose[j][i] = matrix[i][j];
                
            }
        }

        return transpose;
        
    }
}